src/pages/Components/Notification/previews/notification-danger.preview.html

```diff
@@ -0,0 +1,9 @@
+<div class="example">
+  <div class="mc-notification mc-notification-danger">
+    <h3>Danger notification</h3>
+    <p>
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.
+    </p>
+    <a href="#">Link</a>
+  </div>
+</div>
```

Comment:
You have to add classes on your elements.

Reply:
the `role="alert"` should only be used when something pop, for example when there is a error message after a form submit or failed validation, or a server error. We should add this in the docs in semantic and accessibility as a rule, with the related example, but not use it in all examples.

what two expect from an element with a `role="alert"`  :

Screen readers may interrupt current output (whether it's speech or braille) and immediately announce or display the alert message.

we should add this to the docs and with a link to the Mozilla docs : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role